css/css-fonts/parsing/font-face-src-tech.html

```diff
@@ -39,11 +39,14 @@
     { src: 'url("foo.ttf") tech(none)', valid: false },
     { src: 'url("foo.ttf") tech(normal)', valid: false },
     { src: 'url("foo.ttf") tech(xyzzy)', valid: false },
+    { src: 'url("foo.ttf") tech(xyzzy, features-opentype)', valid: false },
+    { src: 'url("foo.ttf") tech(features-opentype, xyzzy)', valid: false },
     // format() function must precede tech() if both are present
     { src: 'url("foo.ttf") format(opentype) tech(features-opentype)', valid: true },
     { src: 'url("foo.ttf") tech(features-opentype) format(opentype)', valid: false },
     // Unsupported technology (for example: no browser has incremental transfer yet), might be
     // dropped from the list, next component of the list should be accepted.
+    { src: 'url("foo.ttf") tech(incremental, color-SVG, features-graphite, features-aat)', dontcomparetech: true, valid: false },
```

Comment:
I'm puzzled why this line says `valid: false` .... surely the `tech()` function accepts a comma-separated list of font technologies, so this should be valid? What am I missing here?

Reply:
@jfkthame Thanks for the review! I've fixed it per https://github.com/w3c/csswg-drafts/issues/8793 discussion